test/integration/driver.go

```diff
@@ -77,20 +73,16 @@ var (
 )
 
 func init() {
-	flag.StringVarP(&params.hub, "hub", "h", "gcr.io/istio-testing",
-		"Docker hub")
-	flag.StringVarP(&params.tag, "tag", "t", "",
-		"Docker tag")
-	flag.StringVar(&params.mixerImage, "mixerImage", "gcr.io/istio-testing/mixer:"+mixerTag,
+	flag.StringVar(&params.hub, "hub", "gcr.io/istio-testing", "Docker hub")
+	flag.StringVar(&params.tag, "tag", "", "Docker tag")
+	flag.StringVar(&params.mixerImage, "mixer", "gcr.io/istio-testing/mixer:"+mixerTag,
 		"Mixer Docker image")
-	flag.StringVarP(&params.namespace, "namespace", "n", "",
+	flag.StringVar(&params.namespace, "n", "",
 		"Namespace to use for testing (empty to create/delete temporary one)")
-	flag.StringVarP(&kubeconfig, "config", "c", "platform/kube/config",
+	flag.StringVar(&params.kubeconfig, "c", "platform/kube/config",
```

Comment:
WE should be following same convention as kubectl.. -kubeconfig, KUBECONFIG and .kube/config